pinc/page_controls.inc

```diff
@@ -35,15 +35,10 @@ function get_proofreading_interface_data_js()
     ];
 
     global $code_url;
-    $button_images = [
-        "imgVSplit" => "$code_url/tools/proofers/gfx/bt4.png",
```

Comment:
we use these icons in 2 other places I think. Should we update all usages to be consistent throughout the site?

Reply:
At least one of the other places is the enhanced interface where it's next to a bunch of other of these old button icons. I wouldn't change anything in that interface and just keep what we have.